packages/trader/package.json

```diff
@@ -24,15 +29,14 @@
     "url": "git+https://github.com/UMAprotocol/protocol.git"
   },
   "files": [
-    "/src/**/*.js"
+    "/dist/**/*.js"
```

Comment:
is this an input our output dir?

Reply:
files (at least as far as I understand) tells the release process what files to include in the npm release.

In this case, it would just be the files compiled by the prod tsconfig, which get dumped in dist. Everything else is for local development if I understand correctly.